Cartfile

```diff
@@ -1,7 +1,7 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.7
-github "mapbox/MapboxDirections.swift" ~> 0.16.0
+github "mapbox/MapboxDirections.swift" "voiceLocale"
```

Comment:
this has been or will be merged to `master`, right?

Reply:
@JThramer waiting https://github.com/mapbox/MapboxDirections.swift/pull/235 before changing this.